src/simularium/VisDataCache.ts

```diff
@@ -242,6 +281,7 @@ class VisDataCache {
         this.tail = null;
         this.numFrames = 0;
         this.size = 0;
+        this.onCacheUpdate();
```

Comment:
idea: would it make sense to just have the changed values sent? 
ie, this.cacheUpdateCallback would be updated only with new values. 
maybe not helpful depending on the way this ends up being set up for the installing app/website